python/tvm/relay/op/nn/nn.py

```diff
@@ -202,6 +203,8 @@ def conv2d(data,
         dilation = (dilation, dilation)
     if isinstance(padding, int):
         padding = (padding, padding)
+    # convert 2-way padding to 4-way padding
+    padding = get_pad_tuple(padding)
```

Comment:
Since we also have conv1d and conv3d, maybe it's better to rename this function to `get_2d_pad_tuple`?

Reply:
in topi/nn/util, we have get_pad_tuple1d and get_pad_tuple3d, so maybe I will rename it as get_pad_tuple2d for consistency